src/nvim/path.c

```diff
@@ -1409,38 +1409,6 @@ static int expand_backtick(garray_T *gap, char *pat, int flags)
   return cnt;
 }
 
-#ifdef BACKSLASH_IN_FILENAME
-/// Replace all slashes by backslashes.
-/// This used to be the other way around, but MS-DOS sometimes has problems
-/// with slashes (e.g. in a command name).  We can't have mixed slashes and
-/// backslashes, because comparing file names will not work correctly.  The
-/// commands that use a file name should try to avoid the need to type a
-/// backslash twice.
-/// When 'shellslash' set do it the other way around.
-/// When the path looks like a URL leave it unmodified.
-void slash_adjust(char *p)
```

Comment:
Why is this removed? Won't that cause backslashes in paths return by libuv to remain backslashes?

Reply:
That assumes that libuv returns backslashes, which afaik is not the case.

Reply:
I just tested with both luv and FFI on Windows. It is the case.